staker-rewards/lib/rewards-calculator.js

```diff
@@ -87,7 +82,7 @@ export default class RewardsCalculator {
     const totalRewardsSum = operatorsRewards.reduce(
       (accumulator, rewards) =>
         accumulator.plus(
-          rewards.totalRewards.toFixed(noDecimalPlaces, BigNumber.ROUND_DOWN)
+          toFormat(rewards.totalRewards, false, BigNumber.ROUND_DOWN)
```

Comment:
I'd rather leave it as it is. We don't need to format here, just `toFixed` in order to perform addition.

Reply:
Why are we doing `toFixed` here? Can't we just add two big ints?

Reply:
We need to crop the decimal part before adding the value to the `accumulator` to keep the results consistent. However seems like `rewards.totalRewards.integerValue(BigNumber.ROUND_DOWN)` fits better here than `          rewards.totalRewards.toFixed(noDecimalPlaces, BigNumber.ROUND_DOWN)` despite the eventual result will be the same.

Reply:
Cool, cool. Thanks for clarification. I'll update this line.